src/wrappers/tensor.rs

```diff
@@ -448,6 +448,40 @@ impl Tensor {
         Ok(Tensor { c_tensor })
     }
 
+    /// Creates a tensor from data that is assumed to be initialized.
+    /// Resize operations are now allowed on this tensor without copying the data first.
+    pub fn f_of_blob(
```

Comment:
This should also be unsafe, right? Since data can be an invalid pointer.

Reply:
yes, I've changed this to be unsafe as well, and added clippy SAFETY docs.